integration/kubernetes/overhead/test_fio/workloads.yaml.in

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fio
+  labels:
+    app: overhead
+spec:
+  runtimeClassName: @RUNTIMECLASS@
+  containers:
+  - name: fio
+    image: egernst/fio
```

Comment:
nack on a private image - we should not be using an image from a private hub account, ideally. I know, sometimes we struggle to find an image to use that is maintained, upto date etc. And, over in our [docker storage metrics](https://github.com/kata-containers/tests/tree/master/metrics/storage), we build our own image based on ubuntu with an apt install of fio...
I have two thoughts here then:
- we could pull ubuntu, and then run the apt command (`exec` it) on that image dynamically.
- we could set up a cluster local image store (it seems one can do that by invoking the docker image store in a container, so it does not seem that hard....), and then build/inject our own images there. I think that is likely a better long term solution, as then we can host our own images and have the dockerfiles local and uploaded, and remove a lot of the requirement on using upstream images.